pyfar/constants/frequency_weighting.py

```diff
@@ -0,0 +1,198 @@
+"""File containing all frequency weighting functions."""
+from typing import Literal, Union
+import numpy as np
+
+# Constants for the weighting curve formulas.
+# See appendix E in IEC 61672-1.
+_F_1 = 20.598997057618316
+_F_2 = 107.65264864304629
+_F_3 = 737.8622307362901
+_F_4 = 12194.217147998012
+_A_1000 = -2
+_C_1000 = -0.062
+
+# Constants for nominal band corrections, taken from table 3 in IEC 61672-1.
+# Could be replaced/combined with a dedicated frequency_bands_nominal()
+# function later.
+_NOMINAL_THIRDBAND_FREQUENCIES = np.array([
```

Comment:
```suggestion
_NOMINAL_THIRD_OCTAVE_BAND_FREQUENCIES = np.array([
```
In english it's either a third octave or simply third, third band would be the third band not a third octave. ;)
I'll not mark this everywhere, but there are a number of occurences in this PR. (It's a very minor comment, indeed)